pydfs_lineup_optimizer/sites/draftkings/classic/importer.py

```diff
@@ -48,14 +55,17 @@ def _row_to_player(self, row):
 
     def import_players(self):  # pragma: no cover
         with open(self.filename, 'r') as csv_file:
-            first_line = csv_file.readline()
-            csv_file.seek(0)
-            if 'TeamAbbrev' in first_line:
-                csv_data = csv.DictReader(csv_file, skipinitialspace=True)
-            else:
-                csv_data = csv.DictReader(islice(csv_file, 7, None),
-                                          skipinitialspace=True)
-            players = [self._row_to_player(row) for row in csv_data]
+            start_line = 0 # Find line with 'TeamAbbrev', that's where players data starts
+            while True:
```

Comment:
If user pass incorrect file without TeamAbbrev program will be in infinity loop. Why do you change this method? From csv samples TeamAbbrev placed in first or eighth row.

Reply:
Its possible that they extend comments and move the beginning of data to line 9 or lower. Searching for the line makes it more resilient to changes.
Another reason is that my scripts do not use line 8.
I'll add the fix for possible infinite loop.